lib/pci.c

```diff
@@ -225,11 +225,15 @@ pci_hdr_write(vfu_ctx_t *vfu_ctx, const char *buf, loff_t offset)
         ret = handle_command_write(vfu_ctx, cfg_space, buf);
         break;
     case PCI_STATUS:
+        /* FIXME ignoring write completely is wrong as some bits are RW1C */
         vfu_log(vfu_ctx, LOG_INFO, "write to status ignored");
         break;
-    case PCI_INTERRUPT_PIN:
-        vfu_log(vfu_ctx, LOG_ERR, "attempt to write read-only field IPIN");
-        ret = ERROR_INT(EINVAL);
+    /* read-only registers */
```

Comment:
Can you add a bit more context as to why we're ignoring the RO writes please? And perhaps which OS's do it?

Reply:
sure